src/core/jpx.js

```diff
@@ -24,20 +24,101 @@ class JpxError extends BaseException {
 }
 
 class JpxImage {
-  static #module = null;
-
-  static decode(data, decoderOptions) {
-    decoderOptions ||= {};
-    this.#module ||= OpenJPEG({ warn });
-    const imageData = this.#module.decode(data, decoderOptions);
-    if (typeof imageData === "string") {
-      throw new JpxError(imageData);
+  static #buffer = null;
+
+  static #handler = null;
+
+  static #instantiationFailed = false;
+
+  static #modulePromise = null;
+
+  static #openjpegWasmUrl = "";
```

Comment:
Please keep the initial value consistent with e.g. DefaultPartialEvaluatorOptions.
```suggestion
  static #openjpegWasmUrl = null;
```